controllers/console/plugin.go

```diff
@@ -50,9 +50,11 @@ const (
 func CreateOrUpdatePlugin(ctx context.Context, cl client.Client, config *rest.Config, namespace string, log logr.Logger) error {
 
 	// check if we are on OCP
-	if isOpenshift, err := utils.IsOnOpenshift(config); err != nil {
-		return errors.Wrap(err, "failed to check if we are on Openshift")
-	} else if !isOpenshift {
+	clusterCapabilities, err := cluster.NewCapabilities(config)
```

Comment:
Is there reason to create new caps here? I would prefer to pass it from main.go.

Reply:
No reason, I just misread the code and thought it was the only way to do it. Now I think I saw the right way